examples/fleet/helloworld/README.md

```diff
@@ -0,0 +1,170 @@
+
+   ```shell
+   wget https://k8s.io/examples/service/load-balancer-example.yaml
+
+   cat <<EOF >./kustomization.yaml
+   namespace: hello-world
+   resources:
+   - load-balancer-example.yaml
+   EOF
+   ```
+
+2. Deploy the application and expose a LoadBalancer service: 
+
+   ```shell
+   kubectl create namespace hello-world
+   kubectl apply -k ./
+   
+   kubectl expose deployment hello-world -n hello-world --type=LoadBalancer --name=my-service
+   ```
+
+3. Verify the application is deployed but not running:
+
+   ```shell
+   kubectl get all -n hello-world
+   ```
+   
+   The output is similar to:
+   
+   ```console
+   NAME                 TYPE           CLUSTER-IP    EXTERNAL-IP   PORT(S)          AGE
+   service/my-service   LoadBalancer   10.0.184.49   <pending>     8080:32614/TCP   4s
+   
+   NAME                          READY   UP-TO-DATE   AVAILABLE   AGE
+   deployment.apps/hello-world   0/5     0            0           28s
+   ```
+
+   Note that the deployment is not deployed (`READY: 0/5`) and the service doesn't have an external IP (`EXTERNAL-IP: <pending>`).
+   This is working as expected as the objective to deploy the application to member clusters not the hub cluster.
+
+### 2. Place to Member Clusters
+
+#### Scenario A: Place to all member clusters
+
+1. Place to all the member clusters ([hello-world-crp-all-clusters.yaml](https://raw.githubusercontent.com/Azure/AKS/master/examples/fleet/helloworld/hello-world-crp-all-clusters.yaml)):
+
+   ```shell
+   kubectl apply -f https://raw.githubusercontent.com/Azure/AKS/master/examples/fleet/helloworld/hello-world-crp-all-clusters.yaml
+   ```
+
+2. Verify the placement status:
+
+   ```shell
+   kubectl get crp hello-world -o yaml | grep status -A 1000
+   ```
+
+   The output is similar to:
+
+   ```yaml
+   status:
+     conditions:
+     - lastTransitionTime: "2022-09-12T23:31:18Z"
+       message: Successfully scheduled resources for placement
+       observedGeneration: 1
+       reason: ScheduleSucceeded
+       status: "True"
+       type: Scheduled
+     - lastTransitionTime: "2022-09-12T23:31:19Z"
+       message: Successfully applied resources to member clusters
+       observedGeneration: 1
+       reason: ApplySucceeded
+       status: "True"
+       type: Applied
+     selectedResources:
+     - group: apps
+       kind: Deployment
+       name: hello-world
+       namespace: hello-world
+       version: v1
+     - kind: Service
+       name: my-service
+       namespace: hello-world
+       version: v1
+     - kind: Namespace
+       name: hello-world
+       version: v1
+     targetClusters:
+     - aks-member-3
+     - aks-member-2
+     - aks-member-1
+   ```
+   
+   All the resources under namespace `hello-world` (including the namespace) are selected and placed to all the member clusters.
+
+3. Verify the application is deployed in the selected clusters:
+
+   ```shell
+   KUBECONFIG=member1 kubectl get all -n hello-world
```

Comment:
this only verifies that it worked on member cluster 1 but the CRP is supposed to place it on 3, right? Should we clarify that?

Reply:
done